index.Rmd

```diff
@@ -4,11 +4,11 @@ author: Willow Flood
 date: Fall 2022 Semester
 ---
 
-# Introduction
+# Introduction 
```

Comment:
It would maybe be a good idea to first give some background information to readers (ie., describe what Peninsula Sandstone Fynbos are) before getting into the problem of the project.